dashboard/test/fixtures/plc_courses.yml

```diff
@@ -0,0 +1,4 @@
+Exploring Computer Science PD Course:
+  name: "Exploring Computer Science"
+  created_at: 2015-10-16 22:36:38.000000000 Z
+  updated_at: 2015-10-16 22:36:38.000000000 Z
```

Comment:
It's not possible to get into this state naturally.

PLC course units get created by scripts. When we are seeding a script we see if any plc objects need to be created by looking at the professional_learning_course attribute on the script. If it's there, then we find_or_create with that name.